pages/osx/carthage.md

```diff
@@ -0,0 +1,23 @@
+# carthage
+
+> Carthage is a dependency management tool for Swift and Objective-C.
+
+- Update the current set of dependencies:
```

Comment:
Umm .. I think you mistakenly put the sentence in the heading. I meant it in the example description. Your heading is fine.